src/main/scala/se/nimsa/sbx/util/ReverseAnonymizationFlow.scala

```diff
@@ -67,7 +67,7 @@ class ReverseAnonymizationFlow() extends GraphStage[FlowShape[DicomPart, DicomPa
             // FIXME: remove println
             println(">>>> grabbed meta, isAnon: " + isAnonymized)
             println(">>>> grabbed meta, canDoReverse: " + canDoReverseAnon)
-            pull(in)
+            push(out, metaPart)
```

Comment:
If we push the DicomMetaPart downstream from here, and not discard, every flow after that needs/should to be aware of the meta part. On the other hand a 'downstream' flow could make use of it then. Not sure what's best.